wsagent/che-wsagent-core/src/main/java/org/eclipse/che/wsagent/server/CheWebSocketEndpoint.java

```diff
@@ -28,6 +43,10 @@
 @ServerEndpoint(value = "/wsagent", configurator = GuiceInjectorEndpointConfigurator.class)
 public class CheWebSocketEndpoint extends BasicWebSocketEndpoint {
```

Comment:
I think it would be better to rename it to make it clear that it is wsagent endpoint